packages/extension-chains/src/index.ts

```diff
@@ -17,6 +17,48 @@ const definitions = new Map<string, MetadataDef>(
   CENNZNetChain.map((def) => [def.genesisHash, def])
 );
 
+export function getLatestMetaFromServer(genesisHashExpected: string) {
+  const xmlHttp = new XMLHttpRequest();
+//xmlHttp.open( "GET", "https://raw.githubusercontent.com/cennznet/api.js/master/packages/api/src/staticMetadata.ts", false ); // false for synchronous request
+  xmlHttp.open("GET", "https://raw.githubusercontent.com/cennznet/api.js/test1/packages/api/src/staticMetadata.ts", false);
+  xmlHttp.send( null );
+  let response = xmlHttp.responseText;
```

Comment:
I think this warrants a structural change on our api.js files
something like
```bash
staticMetadata/
  index.ts
    0x1234...-40.scale
    0x1234...-41.scale
    0x5555...-42.scale
```

and we query for only the one we want to check for